### PR TITLE
Added Guest account test cases

### PR DESCRIPTION
**Pull request by furqanmlk**
#### Summary

* MM-T1371 (1.0) User profile popover shows guest badge

* MM-T1397 (1.0) Guest tag in search in:

* MM-T1369 (1.0) System message when user is added specifies the guest status

* MM-T1360 (1.0) Taken to login screen when removed from last channel

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-32502

### CODE REVIEW

**Comment by josephbaylon** [comment]
@furqanmlk Please fix lint and code block issues

**Review by hahmadia — APPROVED** [review]

File: e2e/cypress/integration/enterprise/guest_accounts/guest_add_spec.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        // * The system message should contain 'added to the channel as a guest'
+        cy.getLastPostId().then((id) => {
+            cy.get(`#postMessageText_${id}`).should('contain', `@${guestUser.username} added to the channel as a guest`);
+        });
+    });
+
+    it('MM-T1397 Guest tag in search in:', () => {
+        demoteGuestUser(guestUser);
+
+        // cy.apiLogin(guestUser);
```
Comment (josephbaylon) [inline e2e/cypress/integration/enterprise/guest_accounts/guest_add_spec.js:76]:
```suggestion
```

**Review by josephbaylon — APPROVED** [review]
LGTM. Just minor cleanup below. Thanks @furqanmlk !!

**Review by prapti — APPROVED** [review]
Overall looks great! T-1369 and T-1371 don't have test scripts in Zephyr, so I didn't exactly know the test steps to validate against, but your steps here makes sense, hence approving! Could you also update the tests in Zephyr?
Thank you @furqanmlk!

**Pull request merged**